PurchasesCoreSwift/Purchasing/RCReceiptFetcher.swift

```diff
@@ -1,48 +0,0 @@
-//
-//  RCReceiptFetcher.swift
```

Comment:
same file, but I just renamed the file from `RCReceiptFetcher` -> `ReceiptFetcher`